ui/src/app/components/apps-panel/expanded-app.tsx

```diff
@@ -41,53 +54,163 @@ const styles = reactStyles({
-  />
-);
+const SettingsButton = (props: { onClick: Function; disabled?: boolean }) => {
+  const { onClick, disabled } = props;
+  return (
+    <AppsPanelButton
+      {...{ onClick, disabled }}
+      icon={faGear}
+      buttonText='Settings'
+    />
+  );
+};
+
+const JupyterButtonRow = (props: {
+  workspace: Workspace;
+  onClickRuntimeConf: Function;
+}) => {
+  const { workspace, onClickRuntimeConf } = props;
+  return (
+    <FlexRow>
+      <SettingsButton onClick={onClickRuntimeConf} />
+      <RuntimeStateButton {...{ workspace }} />
+      <NewNotebookButton {...{ workspace }} />
+    </FlexRow>
+  );
+};
+
+// TODO generalize as UserAppButtonRow?
+const CromwellButtonRow = (props: {
+  userApp: UserAppEnvironment;
+  workspaceNamespace: string;
+}) => {
+  const { userApp, workspaceNamespace } = props;
+  const [launching, setLaunching] = useState(false);
 
-export const ExpandedApp = (props: {
+  return (
+    <FlexRow>
+      <TooltipTrigger
+        disabled={false}
+        content='Support for configuring Cromwell is not yet available'
+      >
+        {/* tooltip trigger needs a div for some reason */}
+        <div>
+          <SettingsButton disabled={true} onClick={() => {}} />
+        </div>
+      </TooltipTrigger>
+      <TooltipTrigger
+        disabled={false}
+        // RW-9304
+        content='Support for pausing Cromwell is not yet available'
+      >
+        {/* tooltip trigger needs a div for some reason */}
+        <div>
+          <AppsPanelButton
+            disabled={true}
+            onClick={() => {}}
+            icon={faPause}
+            buttonText='Pause'
+          />
+        </div>
+      </TooltipTrigger>
+      <TooltipTrigger
+        disabled={!launching && canCreateApp(userApp)}
+        content='A Cromwell app exists or is being created'
+      >
+        {/* tooltip trigger needs a div for some reason */}
+        <div>
+          <AppsPanelButton
+            disabled={launching || !canCreateApp(userApp)}
+            onClick={() => {
+              setLaunching(true);
+              appsApi().createApp(workspaceNamespace, defaultCromwellConfig);
+            }}
+            icon={faPlay}
+            buttonText={launching ? 'Launching' : 'Launch'}
+          />
+        </div>
+      </TooltipTrigger>
+    </FlexRow>
+  );
+};
+
+// TODO: refine and style more like RuntimeStatusIcon
+const AppStatus = (props: { userApp: UserAppEnvironment }) => {
+  const { status } = props.userApp;
+  return <span>{status}</span>;
+};
+
+interface ExpandedAppProps {
   appType: UIAppType;
+  initialUserAppInfo: UserAppEnvironment;
   workspace: Workspace;
   onClickRuntimeConf: Function;
   onClickDeleteRuntime: Function;
-}) => {
+}
+export const ExpandedApp = (props: ExpandedAppProps) => {
   const { runtime } = useStore(runtimeStore);
-  const { appType, workspace, onClickRuntimeConf, onClickDeleteRuntime } =
-    props;
+  const {
+    appType,
+    initialUserAppInfo,
```

Comment:
TODO: keep the user app info updated.  Here we only check at AppsPanel render time.